### ISSUE TITLE
Error Update via OTA Menu

### ISSUE THREAD

**Issue by lucaobermaier** [jomjol/AI-on-the-edge-device#1432]
### The Problem

Hello,

I tried to update via the OTA menu to the current version 13.0.1. - running on 12.0.1.
After hitting the upload and reboot button I always get this message:

"Server closed the connection abruptly!"

Best regards

### Version

12.0.1

### Logfile

```shell
2022-11-29T17:28:57: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
2022-11-29T17:28:57: =============================================================================================
2022-11-29T17:28:57: =================================== Main Started ============================================
2022-11-29T17:28:57: =============================================================================================
2022-11-29T17:28:57: Branch: 'HEAD', Tag: 'v12.0.1', Revision: dfe2c46, Date/Time: 2022-09-29 17:13
2022-11-29T17:52:50: Reboot triggered by Software (5s).
2022-11-29T17:52:50: Reboot in 5sec
2022-11-29T17:53:10: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
2022-11-29T17:53:10: =============================================================================================
2022-11-29T17:53:10: =================================== Main Started ============================================
2022-11-29T17:53:10: =============================================================================================
2022-11-29T17:53:10: Branch: 'HEAD', Tag: 'v12.0.1', Revision: dfe2c46, Date/Time: 2022-09-29 17:13
```

**Comment by jomjol** [comment]
Please try the following:
1) extraxt the firmware.bin from the zip-file and only use this for the update
2) after reboot, there might be a error message about a mismatch between firmware and html --> ignore it
3) do the update again with the full update.zip

That should work.

**Comment by lucaobermaier** [comment]
same issue.

**Comment by caco3** [comment]
The OTA happens with the still running 12.0.1.
Therefore it can't be a new bug.

**Comment by lucaobermaier** [comment]
sorry - i dont get the last answer.
which steps can i try next?
i already made flashed the device completly new but i always get the same error

**Comment by caco3** [comment]
What I wanted to say:
When you do an OTA update, you upload the new firmware to the device. The old firmware then is still running (12.0.1 in your case) and performs the erasing of the flash and writing of the new firmware (13.0.1 in your case) to the flash.
Only after that completes and you reboot, the new firmware is active.

Since you see those issues before the update completed, the are caused by the 12.0.1.
And we knew there where update problems. And since they are hopefully fixed in 13.0.1, this issue no longer will be an issue once you succeeded to flash 13.0.1.

**Comment by lucaobermaier** [comment]
flash update worked properly like always
